02Lesson/01Homework/index.html

```diff
@@ -1,8 +1,129 @@
 <!DOCTYPE html>
 <html>
 <head>
-  <title></title>
+  <title>All About Brady</title>
+  <meta charset="utf-8">
 </head>
-<body>
+
+<body style="background-color:lightblue">
+  <h1 style="text-align:center">
+    MoBrady MoProblems
+  </h1>
+  <p style="text-align:center">
+    <em>You can sleep when you're dead</em>
+  </p>
+    <div style="text-align:center">
+      <img src="https://scontent-dft4-1.xx.fbcdn.net/v/t1.0-9/10262186_10204371733316536_8640703755058897068_n.jpg?oh=4d29b309382bed855da92ee7ed585681&oe=5891FF36" alt="Disco Ball" height="250" width="250"/>
+    </div>
+  <h2 style="font-family:fantasy">
```

Comment:
Indentation: After closing `div` tag, the following opening tag needs to be on the same level